exercises/practice/connect/src/test/java/ConnectTest.java

```diff
@@ -0,0 +1,220 @@
+import org.junit.Ignore;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class ConnectTest {
+
+    @Test
+    public void anEmptyBoardHasNoWinner() {
+
+        //GIVEN
+        var board = new String[]{
+            ". . . . .",
+            " . . . . .",
+            "  . . . . .",
+            "   . . . . .",
+            "    . . . . ."
+        };
+        Connect cut = new Connect(board);
+
+        //WHEN
+        var winner = cut.computeWinner();
+
+        //THEN
+        assertEquals(Connect.Winner.NONE, winner);
```

Comment:
```suggestion
        assertThat(winner).isEqualTo(Connect.Winner.NONE);
```

An example of using AssertJ